addons/open-in-gandi/styles.css

```diff
@@ -0,0 +1,24 @@
+.preview .sa-gd-button {
+    padding: 0px 5px 0px 11px;
+    margin-top: 0;
+    border-radius: 0.25rem;
+    height: 2.5rem;
+    font-size: 0.875rem;
+  }
+  .preview .sa-gd-button:not(.scratch) {
+    background: #0094ff;
+  }
+
+  .preview .sa-gd-button:before {
+    background-image: url(https://super-static-assets.s3.amazonaws.com/c0789e05-1ed3-46e5-9c10-baaa8afbde56/uploads/favicon/d3318337-0ffd-4687-882e-31814beb6085.png);
```

Comment:
You uploaded `gandi-logo.png` but you're still using amazonaws.com for image hosting.

Reply:
yeah...ig i just delete it

Reply:
No, use the Gandi logo via Scratch Addons. Using another image hosting service would require a change to the privacy policy.

Reply:
oh